tests/integration/test_cli.py

```diff
@@ -262,29 +262,6 @@ def test_install_parse_error(PipenvInstance):
         assert 'u/\\/p@r$34b13+pkg' not in p.pipfile['packages']
 
 
-@pytest.mark.code
-@pytest.mark.check
-@pytest.mark.unused
-@pytest.mark.skip_osx
-@pytest.mark.needs_internet(reason='required by check')
-def test_check_unused(PipenvInstance):
-    with PipenvInstance(chdir=True) as p:
-        with open('__init__.py', 'w') as f:
-            contents = """
-import click
-import records
-import flask
-            """.strip()
-            f.write(contents)
-        p.pipenv('install requests click flask')
-
-        assert all(pkg in p.pipfile['packages'] for pkg in ['requests', 'click', 'flask']), p.pipfile["packages"]
-
-        c = p.pipenv('check --unused .')
-        assert 'click' not in c.stdout
-        assert 'flask' not in c.stdout
```

Comment:
Notice that it doesn't check that `requests` is actually in the `c.stdout` because if it had done that, it would have been obvious this hasn't been working in ages.